lib/ratchet/eex.ex

```diff
@@ -12,11 +12,23 @@ defmodule Ratchet.EEx do
   @doc """
   Build an EEx statement fetching content
 
-      iex> Ratchet.EEx.eex_content("lolwat")
-      "<%= Ratchet.Data.content(lolwat) %>"
+      iex> Ratchet.EEx.eex_content("lolwat", ["Content"])
+      [
+        "<%= if Ratchet.Data.content?(lolwat) do %>",
+        "<%= Ratchet.Data.content(lolwat) %>",
+        "<% else %>",
+        "Content",
+        "<% end %>",
+      ]
   """
-  def eex_content(property) do
-    "<%= Ratchet.Data.content(#{property}) %>"
+  def eex_content(property, default) do
+    [
+      "<%= if Ratchet.Data.content?(#{property}) do %>",
+      "<%= Ratchet.Data.content(#{property}) %>",
+      "<% else %>",
+      default,
+      eex_close,
+    ] |> List.flatten
```

Comment:
Would sort of rather "splice" the default content into the list rather than having to flatten. Not sure if that's doable? Perhaps could use `unquote_splicing`, but doesn't feel worth converting this to a macro.